### PR TITLE
add varibles to changes unit and env vars for dnsdist service

### PR DESCRIPTION
**Pull request by SrX**
- add variables to changes unit and env vars for dnsdist service

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ By default, dnsdist is installed from the software repositories configured on th
       dnsdist_install_repo: "{{ dnsdist_powerdns_repo_13 }}"
 ```
 
-The examples above, show how to install dnsdist from the official PowerDNS repositories
+The examples above, show how to install DNSdist from the official PowerDNS repositories
```
Comment (Habbie) [inline README.md:42]: why this change?

**Review by Habbie — COMMENTED** [review]
one question. Code changes seem fine (but I don't use Ansible much)

File: README.md (hunk 1 above)
Comment (SrX) [reply README.md:42]: just to have DNSdist written in same way in the readme file, at this moment it use two ways: DNSdist and dnsdist. I think the right one would be dnsdist, as it's opensource roles. I'll do a another MR later for this.

Comment (Habbie) [reply README.md:42]: :+1:

**Review by npmdnl — COMMENTED** [review]
LGTM
